spec.bs

```diff
@@ -221,6 +224,35 @@ The <dfn for="interest group">estimated size</dfn> of an [=interest group=] |ig|
     [=interest group ad/render url=].
   1. The [=string/length=] of |ad|'s [=interest group ad/metadata=] if the field is not null.
 
+<div algorithm="check interest group permissions">

```

Comment:
The algorithm attribute text isn't needed here since it will adopt the text of the first definition below.

Same goes with the below `<div algorithm>` further down